cw-orch-daemon/src/queriers/rpc/node.rs

```diff
@@ -11,6 +11,7 @@ use prost::Message;
 
 /// Querier for the Tendermint node.
 /// Supports queries for block and tx information
+/// @TODO: all tendermint queries should use the tendermint-rpc explicitly instead of hitting the tendermint node with the typed queries.
```

Comment:
Outstanding issue

Reply:
what do you mean ? The tendermint rpc library does not expose those types, so we can't really do that can we ?

Reply:
Only when it does expose those types - I think most of them are covered in this PR but there could still be some outstanding ones.